lib/improver/uv_index.py

```diff
@@ -34,19 +34,19 @@
 from cf_units import Unit
 
 
-def calculate_uv_index(uv_upward, uv_downward, scale_factor):
+def calculate_uv_index(uv_upward, uv_downward, scale_factor=3.6):
     """
     A plugin to calculate the uv index using radiation flux in UV downward
     at surface, radiation flux UV upward at surface and a scaling factor.
     The scaling factor is configurable by the user.
 
     Args:
         uv_upward (iris.cube.Cube):
-            A cube of the radiation flux in UV upward at surface (W/m^2)
+            A cube of the radiation flux in UV upward at surface (W m-2)
         uv_downward (iris.cube.Cube):
-            A cube of the radiation flux in UV downward at surface (W/m^2)
+            A cube of the radiation flux in UV downward at surface (W m-2)
         scale_factor (float):
-            The uv scale factor (no units)
+            The uv scale factor. Default is 3.6 (no units)
```

Comment:
Previously we decided that we wouldn't include the default scale factor within the plugin as it wasn't defined there. So this part may not be required (if we are setting our scale factor elsewhere).